.travis.yml

```diff
@@ -1,10 +1,16 @@
+sudo: false
+
 language: ruby
+
+bundler_args: ''
```

Comment:
We have this everywhere. I think that using bundler_args doesn't kill caching, having a custom bundle install does.